spec/lib/solidus_kustomer/client_spec.rb

```diff
@@ -14,6 +14,32 @@
     end
   end
 
+  describe '#create_customer' do
+    it 'creates the customer' do
+      user = create(:user_with_addresses, id: 500)
+      kustomer = described_class.new(api_key: 'my_api_key')
+
+      VCR.use_cassette('create_customer') do
+        response = kustomer.create_customer(user)
+        expect(response['attributes']['externalId']).to eq(user.id.to_s)
+      end
+    end
+  end
+
+  describe '#update_customer' do
+    it 'updates the existing customer information on Kustomer' do
+      user = create(:user, kustomer_id: 'the_customer_UUID')
+      kustomer = described_class.new(api_key: 'my_api_key')
+
```

Comment:
This blank line should not be here, since the line below is also part of the "setup" phase.